cloud/azure/bin/deploy

```diff
@@ -3,3 +3,5 @@
 KEEP_ORIGINAL_PWD=true source "${PROJECT_BASE_DIR}/bin/lib.sh"
 source "${PROJECT_BASE_DIR}/cloud/shared/bin/lib.sh"
 source "${PROJECT_BASE_DIR}/cloud/azure/bin/lib.sh"
+
+# TODO: implement actual Azure deployment logic
```

Comment:
question: is this where we would validate all the azure-specific variables?

Reply:
yeah, this is where the "figure out which code to execute" dance ends and we actually get down to business, probably starting with config validation